src/Service.Tests/SqlTests/RestApiTests/Put/MsSqlPutApiTests.cs

```diff
@@ -233,6 +259,109 @@ public async Task PutOneInViewBadRequest()
             await base.PutOneInViewBadRequest(expectedErrorMessage);
         }
 
+        /// <summary>
+        /// Test to validate successful execution of a request when a rowversion field is missing from the request body.
+        /// In such a case, we don't attempt to NULL out rowversion field (as per PUT semantics) but instead skip updating/inserting the field. 
+        /// </summary>
+        [TestMethod]
+        public async Task PutOneWithRowversionFieldMissingFromRequestBody()
+        {
+            // Validate successful execution of a PUT update when a rowversion field (here 'row_version')
+            // is missing from the request body. Successful execution of the PUT request confirms that we did not
+            // attempt to NULL out the 'row_version' field.
```

Comment:
`PutOneUpdateWithRowversionFieldMissingFromRequestBody` does not include rowversion in the verification query.. so there's no validating whether row_version is touched.

Reply:
https://github.com/Azure/data-api-builder/pull/1596#discussion_r1322397055